docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
 
   gitlab:
     build: gitlab/
+    volumes-from: gitlab-data
```

Comment:
If this is the expectation, then `gitlab-data` (the data container) should be integrated into the `docker-compose.yml` file so that the workflow is nice. Another solution is to just `ADD data.tar /` in the dockerfile for the gitlab image (this works, I've tested it).